components/prism-concurnas.js

```diff
@@ -0,0 +1,38 @@
+Prism.languages.concurnas = {
+	'comment': [
+		{
+			pattern: /(^|[^\\])\/\*[\s\S]*?(?:\*\/|$)/,
+			lookbehind: true
+		},
+		{
+			pattern: /(^|[^\\:])\/\/.*/,
+			lookbehind: true,
+			greedy: true
+		}
+	],
+	'string': {
+		pattern: /(?:[rub]|rb|br)?(\|\||r"|r'|"|')(?:\\.|(?!\1)[^\\\r\n])*\1/i,
```

Comment:
So `rbr"r"` a valid string?

In the [string section](https://concurnas.com/docs/types.html#strings), I could only find `r"|r'|"|'` and `s"|s'` which isn't supported. Is `||foo||` really valid?
Also, is the `r`/`s` prefix really case-insensitive?
```suggestion
		pattern: /[rs]?("|')(?:\\.|(?!\1)[^\\\r\n])*\1/,
```

Reply:
Thanks for this. ||foo|| on it's own is not valid, but the likes of: mylang ||foo|| are. I have added a langext rule to support this. Also I have added code to support 'code in strings'